frontend/src/app/main/ui/hooks/resize.cljs

```diff
@@ -0,0 +1,105 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) UXBOX Labs SL
+
+(ns app.main.ui.hooks.resize
+  (:require
+   [app.common.geom.point :as gpt]
+   [app.common.logging :as log]
+   [app.main.refs :as refs]
+   [app.util.dom :as dom]
+   [app.util.storage :refer [storage]]
+   [rumext.alpha :as mf]))
+
+(log/set-level! :warn)
+
+(def last-resize-type nil)
+
+(defn set-resize-type! [type]
+  (set! last-resize-type type))
+
+(defn use-resize-hook
+  [key initial min-val max-val axis negate? resize-type]
+
+  (let [current-file-id (mf/deref refs/current-file-id)
+        size-state (mf/use-state (or (get-in @storage [::saved-resize current-file-id key]) initial))
+        parent-ref (mf/use-ref nil)
+
+        dragging-ref (mf/use-ref false)
+        start-size-ref (mf/use-ref nil)
+        start-ref (mf/use-ref nil)
+
+        on-pointer-down
+        (fn [event]
```

Comment:
You probably want use here the `mf/use-callback` or `mf/use-fn` (short alias of the first one)
The same as  for the next function.

Reply:
Ok